ui/src/layouts/Group.vue

```diff
@@ -30,6 +30,7 @@ export default {
     },
     computed: {
         columns () {
+            console.log(this.group.width)
```

Comment:
```suggestion
```

Reply:
Fixed the above as Joe is not available today